package.json

```diff
@@ -7,10 +7,14 @@
   },
   "dependencies": {
     "nan": "2.x",
-    "tar": "2.x"
+    "tar": "2.x",
+    "nodereport": "1.x",
```

Comment:
you added these by hand? if you used npm, they would be sorted alphabetically, and if the next person who adds a dep uses npm, it will sort them all, creating `git diff` churn, so its better to follow npm convention and keep them sorted.